src/components/RadioButton/RadioButton-story.js

```diff
@@ -9,15 +9,27 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
-import { withKnobs, boolean, text } from '@storybook/addon-knobs';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
 import RadioButton from '../RadioButton';
 import RadioButtonSkeleton from '../RadioButton/RadioButton.Skeleton';
 
+const labelPositions = {
+  'Left (left)': 'left',
+  'Top (top)': 'top',
+  'Right (right)': 'right',
+  'Bottom (bottom)': 'bottom',
```

Comment:
If we are no longer supporting labels to be at the top and bottom on v10 do we want to still add this or can these be removed for v10 ?

Reply:
@aledavila Thank you for pointing this out! Fixed.